### ISSUE TITLE
Missing I18n?

### ISSUE THREAD

**Issue by Tokeli** [Aang23/UndergroundBiomesConstructs#10]
MC: 1.12.2
Forge: 14.23.4.2760
UBC: 1.12-1.2.9

When trying to export blocks with CraftTweaker I ran into this minor issue:

```
[04:36:38] [Server thread/WARN] [minecraft/CommandHandler]: Couldn't process command: ct blocks
java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
        at exterminatorjeff.undergroundbiomes.common.block.wall.UBStoneWall.getLocalizedName(UBStoneWall.java:75) ~[UBStoneWall.class:?]
        at crafttweaker.mc1120.block.MCBlockDefinition.getDisplayName(MCBlockDefinition.java:39) ~[MCBlockDefinition.class:?]
        at crafttweaker.mc1120.commands.Commands$2.executeCommand(Commands.java:80) ~[Commands$2.class:?]
        at crafttweaker.mc1120.commands.CTChatCommand.execute(CTChatCommand.java:71) ~[CTChatCommand.class:?]
        at net.minecraft.command.CommandHandler.tryExecute(CommandHandler.java:119) [bj.class:?]
        at net.minecraft.command.CommandHandler.executeCommand(CommandHandler.java:91) [bj.class:?]
        at net.minecraft.network.NetHandlerPlayServer.handleSlashCommand(NetHandlerPlayServer.java:958) [pa.class:?]
        at net.minecraft.network.NetHandlerPlayServer.processChatMessage(NetHandlerPlayServer.java:937) [pa.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.processPacket(SourceFile:37) [la.class:?]
        at net.minecraft.network.play.client.CPacketChatMessage.processPacket(SourceFile:9) [la.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [hv$1.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_171]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_171]
        at net.minecraft.util.Util.runTask(SourceFile:529) [h.class:?]
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:723) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:397) [nz.class:?]
        at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.resources.I18n
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_171]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_171]
        ... 19 more
Caused by: java.lang.NullPointerException
```

**Comment by Aang23** [comment]
Could I see the relevant CraftTweaker script ?

**Comment by Tokeli** [comment]
Sorry, there was no script, that was attempting to run the `ct blocks` command to dump all block IDs. The crafttweaker dev meanwhile pointed out that it's clientside code on the server: https://github.com/CraftTweaker/CraftTweaker/issues/644

**Comment by Aang23** [comment]
Oh ! Ok. I'm gonna fix that.
This is the kind of issues I keep fixing since since I'm working on UBC.

**Comment by Aang23** [comment]
Fixed in dev, will be out in 1.3.0.
